stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -231,13 +234,33 @@
         period: uint,
         ;; A string representing the function where this authorization is valid. Either
         ;; `stack-stx`, `stack-extend`, or `agg-commit`.
-        topic: (string-ascii 12),
+        topic: (string-ascii 14),
```

Comment:
Just out of curousity...why does the topic string need to increase in length from 12 to 14?

Reply:
It was only to allow for the whole string `stack-increase`, but definitely not strictly necessary. But this line points out that I need to update some of the documentation in this file (like this line).

Reply:
Ah yeah this makes sense. With the comment updated, totally get it.